src/primitives/popover/popover.tsx

```diff
@@ -89,7 +89,20 @@ export const Popover = memo(
       constrainSize = false,
       content,
       disabled,
-      fallbackPlacements,
+      fallbackPlacements = [
+        'top',
+        'top-start',
+        'top-end',
+        'bottom',
+        'bottom-start',
+        'bottom-end',
+        'left',
+        'left-start',
+        'left-end',
+        'right',
+        'right-start',
+        'right-end',
+      ],
```

Comment:
I'd move this value to a `DEFAULT_FALLBACK_PLACEMENTS` variable outside of the component scope and do
```suggestion
      fallbackPlacements = DEFAULT_FALLBACK_PLACEMENTS,
```